rewrite-core/src/main/java/org/openrewrite/internal/StringUtils.java

```diff
@@ -369,4 +372,26 @@ public static String repeat(String s, int count) {
         System.arraycopy(multiple, 0, multiple, copied, limit - copied);
         return new String(multiple);
     }
+
+    private static boolean matchesGlob(@Nullable String value, @Nullable String globPattern) {
+        if ("*".equals(globPattern)) {

```

Comment:
_Should_ `matchesGlob(null, "*")` evaluate to `true`? 🤔

Reply:
That's how I'm treating it currently where I'm using it in rewrite-kubernetes. It does bear discussing that this will create a precedent we will need to be consistent with in other places. Not sure that's always the case now.